sdk/src/types/block/semantic/error.rs

```diff
@@ -40,146 +40,148 @@ pub enum TransactionFailureReason {
     None = 0,
     #[display(fmt = "transaction was conflicting and was rejected")]
     ConflictRejected = 1,
+    #[display(fmt = "transaction was orphaned")]
+    Orphaned = 2,
```

Comment:
We could just remove the manually specified values except for 255, since rust automatically counts up from 0. it would help if this happens again 🤷‍♀️

Reply:
I'll do this next time it happens